packages/wonder-blocks-dropdown/src/components/dropdown-core-virtualized.js

```diff
@@ -59,7 +59,7 @@ type State = {|
 /**
  * Maximum visible items inside the dropdown list
  */
-const MAX_VISIBLE_ITEMS = 10;
+const MAX_VISIBLE_ITEMS = 9;
```

Comment:
One more thing that'd be nice to add is explaining why we set 9 items here. It was a bit tricky to identify why the focused element didn't appear on the screen at some point.